src/App.tsx

```diff
@@ -8,11 +9,24 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

Comment:
it's hardcode. If id will be changed, app will not work as expected.
Use id of the first tab from the array